src/test/app/PayChan_test.cpp

```diff
@@ -832,6 +832,180 @@ struct PayChan_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testDepositAuthCreds()
+    {
+        testcase("Deposit Authorization with Credentials");
+        using namespace jtx;
+        using namespace std::literals::chrono_literals;
+
+        const char credType[] = "abcde";
+
+        auto const alice = Account("alice");
+        auto const bob = Account("bob");
+        auto const carol = Account("carol");
+        Account const dillon{"dillon"};
+        Account const zelda{"zelda"};
```

Comment:
Why the different methods of `Account` creation?

Reply:
fixed